velox/connectors/hive/HiveConfig.cpp

```diff
@@ -152,6 +152,14 @@ bool HiveConfig::isPartitionPathAsLowerCase(const Config* session) const {
   return session->get<bool>(kPartitionPathAsLowerCaseSession, true);
 }
 
+bool HiveConfig::ignoreMissingFiles(const Config* session) const {
+  if (session->isValueExists(kIgnoreMissingFilesSession)) {
```

Comment:
This pattern of double checking (exists + get) is an anti-pattern. Let's change to `session->get<T>(name, defaultValue)`. Would you make a separate PR for that? CC: @kewang1024

Reply:
Updated, thanks.

Reply:
Sure.

Reply:
https://github.com/facebookincubator/velox/pull/8633, thanks.